### DISCUSSION TITLE
Home Assistant example configurations?

### DISCUSSION THREAD

**Discussion by dwareb** [revk/xESP32-Faikout#379] [General]
So, I've setup Home Assistant for the first time today, and have spent the last few hours pouring over documentation on how to get Home Assistant integration setup, MQTT working, etc. I've been making progress, but its quite slow. What could really help is if someone has an example of how they've got their faikin entity setup in that HA configuration.yaml (or mqtt.yaml... you know what I mean). I understand that everyone's configurations for this are going to be different, but I'm reinventing the wheel, and I'm not sure if mine is square. For example, I think its supposed to be setup as a climate device? Should it be setup as multiple devices? But I'm not even positive if that's the best way. Am I missing some fairly simple integration that already exists, or does it need to be written from scratch?

Basically I think it would be a big help to at least see how its been done before, so I can have a starting point. Or if anyone has tips on good learning resource / example projects that would be helpful, I'm all ears.

**Comment by revk** [comment]
Not wishing to be thick, but surely just point HA and the Faikin at the same MQTT server and the Faikins all configure and appear in HA. No manual config actually needed.

**Comment by dwareb** [comment]
Yep, I'm being thick. I thought that it'd be a good idea to try some integration and setup while the Faikin is still on the bench top. Which I see now, was quite bone headed. The automatic discovery isn't sent unless its actually attached to an AC head. Which I've done now, and it's working fine. My mistake!

**Reply by revk** [reply]
Ah, yes, sorry. It does not know what you have and hence what options to offer to HA.
